src/main/flight/pid.c

```diff
@@ -180,9 +190,9 @@ void pidSetItermAccelerator(float newItermAccelerator)
     itermAccelerator = newItermAccelerator;
 }
 
-float pidItermAccelerator(void)
+bool pidOSDAntiGravityActive(void)
```

Comment:
camelCase is normally done by only capitalising the first letter of abbreviations, i.e. `pidOsdAntiGravityActive()`.